pcdsdevices/targets.py

```diff
@@ -260,3 +267,795 @@ def move(self, nxspaces, nyspaces, wait=False):
 
         self.x.mv(xpos, wait=wait)
         self.y.mv(ypos, wait=wait)
+
+    def tweak(self):
+        """
+        Call the tweak function from `pcdsdevice.interface`.
+
+        Use the Left arrow to move x motor left.
+        Use the Right arrow to move x motor right.
+        Use the Down arrow to move y motor down.
+        Use the Up arrow to move y motor up.
+        Use Shift & Up arrow to scale*2.
+        Use Shift & Down arrow to scale/2.
+        Press q to quit.
+        """
+        tweak_base(self.x, self.y)
+
+    # TODO: maybe add an option here for setting parameters directly
+    def set_presets(self):
```

Comment:
These presets are only used in the subclass, I'm not sure if these methods belong in the parent

Reply:
I placed them in the `XYGridTarget` object in 393ee51